clock_exercise/main.qml

```diff
@@ -0,0 +1,43 @@
+import QtQuick 2.12
+import QtQuick.Window 2.12
+import QtQuick.Controls 2.15
+
+Window {
+    width: 480
+    height: 800
+    maximumHeight: height
+    maximumWidth: width
+    minimumHeight: height
+    minimumWidth: width
+    visible: true
+    title: qsTr("Clock Exercise")
+
+    StackView {
+        id: stack
+        initialItem: clockPage
+        anchors.fill: parent
+    }
+
+    Component {
+        id: clockPage
+        Clock {
+            onTimerButtonClicked: stack.push(timerSettingPage)
+            onAlarmButtonClicked: stack.push(alarmPage)
+        }
+    }
+
+    Component {
```

Comment:
Secondo me era presto per aggiungere queste pagine, anche se vuote.
Considera sempre quali sono lo scopo e il contesto della issue che stai svolgendo: se stiamo lavorando sulla homepage dell'orologio, concentriamo lo sviluppo solo su quella.
Nel momento in cui ci servirà imbardare un po' di struttura per eseguire il cambio pagina, apriremo una issue apposita, oppure lo faremo nel momento in cui servirà effettivamente aprire la pagina degli allarmi nella issue in cui è stata sviluppata.

Chiaramente in questo caso non è un problema, ma immagina su un progetto con 8 sviluppatori che macinano 3-4 issue al giorno a testa: cosa succederebbe se ciascuno inserisse dentro le proprie Pull Request anche sviluppi di cose non del tutto pertinenti, per quanto analoghe?
Lo so che in questo caso non ci sono altri 7 sviluppatori insieme a te, ma fai finta che ci siano. Questo tipo di esercizi serve anche a simulare un po' uno stile di lavoro in un ambiente più complesso.